scripts/send-lz-nft.ts

```diff
@@ -0,0 +1,86 @@
+// We require the Hardhat Runtime Environment explicitly here. This is optional
+// but useful for running the script in a standalone fashion through `node <script>`.
+//
+// When running the script with `npx hardhat run <script>` you'll find the Hardhat
+// Runtime Environment's members available in the global scope.
+import { ethers, getNamedAccounts} from "hardhat";
```

Comment:
it would be great, if you write a one-two sentences of what this script does, and why is needed. Same for prepare script